project.godot

```diff
@@ -43,9 +43,10 @@ editor/translations/update_pot_files_automatically=false
 
 [display]
 
-window/size/viewport_width=1920
-window/size/viewport_height=1080
+window/size/viewport_width=1028
+window/size/resizable=false
 window/stretch/mode="viewport"
+window/stretch/aspect="ignore"
```

Comment:
Please do not change project-wide settings, such as the viewport settings here, if you would like your StoryQuest to ultimately be part of the official version of the game.